website/docs/d/vapp_vm.html.markdown

```diff
@@ -93,8 +93,16 @@ The following arguments are supported:
 * `catalog_name` -  The catalog name in which to find the given vApp Template
 * `template_name` -  The name of the vApp Template to use
 * `memory` -  The amount of RAM (in MB) allocated to the VM
+* `memory_reservation` - The amount of RAM (in MB) reservation on the underlying virtualization infrastructure
+* `memory_priority` - Pre-determined relative priorities according to which the non-reserved portion of this resource is made available to the virtualized workload. Values can be: `LOW`, `NORMAL`, `HIGH` and `CUSTOM`
+* `memory_shares` - Custom priority for the resource in MB
+* `memory_limit` - The limit (in MB) for how much of memory can be consumed on the underlying virtualization infrastructure. This is only valid when the resource allocation is not unlimited
 * `cpus` -  The number of virtual CPUs allocated to the VM
 * `cpu_cores` -  The number of cores per socket
+* `cpu_reservation` - The amount of Mhz reservation on the underlying virtualization infrastructure
```

Comment:
Please change case from `Mhz` to `MHz` in all user-facing messages of this PR (docs, parameter descriptions, etc).

Reply:
updated